src/main/java/org/spongepowered/common/data/util/NbtDataUtil.java

```diff
@@ -62,6 +62,9 @@
     public static final String ITEM_BOOK_TITLE = "title";
     public static final String ITEM_BOOK_AUTHOR = "author";
     public static final String ITEM_BOOK_RESOLVED = "resolved";
+    
```

Comment:
I saw that the previous tags had spaces between them to put them in groups, so I did the same.